curp/tests/it/read_state.rs

```diff
@@ -23,6 +23,7 @@ async fn read_state() {
             TestCommandResult::default(),
         );
     });
+    sleep_millis(10).await;
```

Comment:
Why do you sleep here?

Reply:
In order to ensure that the `put_cmd` is proposed before `fetch_read_state`